web/app/src/components/homepage/footer-section.vue

```diff
@@ -86,7 +86,7 @@
   }
   .footer-section-container {
     display: flex;
-    align-items: center;
+    justify-content: center;
```

Comment:
one day i will understand flex.

Reply:
align-items is vertical
justify-content is horizontal XD

You missed a golden opportunity of commenting 
> Weird flex but ok
> \*merges PR\*

Reply:
LOL. Good one.